README.md

```diff
@@ -29,159 +29,26 @@ Afterwards copy the consumer key and secret from OpenStreetMap into your configu
 
 **Run the Tasking Manager**
 
-The easiest way to run the Tasking Manager requires [Docker](https://docs.docker.com/get-started/) and [Docker Compose](https://docs.docker.com/compose/) to be installed on your system.  Afterwards you'll just need:
+The **easiest way** to run the Tasking Manager requires [Docker](https://docs.docker.com/get-started/) and [Docker Compose](https://docs.docker.com/compose/) to be installed on your system.  Afterwards you'll just need:
 
 * One command to get everything together and start the Tasking Manager: `docker-compose up -d`
 * Visit with your browser `http://127.0.0.1:5000`
 
-
 For stopping this command do the job: `docker-compose stop`
 And you can check the logs with `docker-compose logs -f`
 
-Alternatively you can install a [development setup](./README.md#development-setup).
+**Alternatively** you can review how to install a [development setup](./docs/setup-development.md) or deploy a [live installation](./docs/setup-live.md).
```

Comment:
It's missing the `setup-live.md` file

Reply:
Thanks for noticing. Removed for now, as we deprecated [TM3-Production-with-Nginx-Setup-Guide](https://github.com/hotosm/tasking-manager/wiki/TM3-Production-with-Nginx-Setup-Guide).